client/src/component/facebookLoginButton/facebookLoginButton.js

```diff
@@ -18,10 +20,10 @@ class FacebookLoginButton extends React.Component {
         isMobile={false}
         fields="name,picture"
         callback={responseFacebook}
-        render={(renderProps) => (
-          <button className="fb-login-button" onClick={renderProps.onClick}>
+        render={() => (
```

Comment:
Why did u change this? Also you need to include the `onClick`

Reply:
so we use mui components :,) there's also some styling already applied by the mui so it's more consistent